DataLayer/BatteryData/BatteryData.cpp

```diff
@@ -27,173 +27,303 @@
 #include <QDebug>
 
 BatteryData::BatteryData()
-: mod0PcbTemperature_(-1)
-, mod0CellTemperature_(-1)
-, mod0CellVoltages_({-1, -1, -1, -1, -1, -1, -1, -1})
-, mod1PcbTemperature_(-1)
-, mod1CellTemperature_(-1)
-, mod1CellVoltages_({-1, -1, -1, -1, -1, -1, -1, -1})
-, mod2PcbTemperature_(-1)
-, mod2CellTemperature_(-1)
-, mod2CellVoltages_({-1, -1, -1, -1, -1, -1, -1, -1})
-, mod3PcbTemperature_(-1)
-, mod3CellTemperature_(-1)
-, mod3CellVoltages_({-1, -1, -1, -1, -1, -1, -1, -1})
-, batteryVoltage_(-1)
-, batteryCurrent_(-1)
+    : packageID_(-1)
+    , alive_(-1)
+    , packSocAmpHours_(-1)
+    , packSocPercentage_(-1)
+    , packBalanceSoc_(-1)
+    , packBalanceSocPercentage_(-1)
+    , chargingCellVoltageError_(-1)
+    , cellTemperatureMargin_(-1)
+    , dischargingCellVoltageError_(-1)
+    , totalPackCapacity_(-1)
+    , prechargeContactorDriverStatus_(-1)
+    , prechargeState_(-1)
+    , prechargeTimerElapsed_(-1)
+    , prechargeTimerCount_(-1)
+    , lowestCellVoltage_(-1)
+    , lowestCellVoltageNumber_(-1)
+    , highestCellVoltage_(-1)
+    , highestCellVoltageNumber_(-1)
+    , lowestCellTemperature_(-1)
+    , lowestCellTemperatureNumber_(-1)
+    , highestCellTemperature_(-1)
+    , highestCellTemperatureNumber_(-1)
+    , voltage_(-1)
+    , current_(-1)
+    , fan0Speed_(-1)
+    , fan1Speed_(-1)
+    , fanContactors12VCurrentConsumption_(-1)
+    , cmu12VCurrentConsumption_(-1)
+    , bmsCanLockedOut_(-1)
+
```

Comment:
@enochtsang  Some of these are unsigned, should they be initialised to something different?